utils/gdrivefile_util.py

```diff
@@ -76,6 +76,40 @@ def download_file(drive_service, file_name, path_to_download):
     except Exception as ex:
         print(ex)
 
+def upload_file(file_name, drive_service, folder_name, file_path, file_mime_type):
+    # Search for folder id in Drive
+    page_token = None
+    folders = []
+    while True:
+        response = drive_service.files().list(
+            q=f"mimeType='application/vnd.google-apps.folder' and name='{folder_name}'",
```

Comment:
can you try with title={folder_name} instead of name?

Reply:
Solved. Was due to permission scope not including metadata